src/Microsoft.Health.Fhir.Shared.Web/CloudRoleNameTelemetryInitializer.cs

```diff
@@ -0,0 +1,18 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Microsoft.ApplicationInsights.Channel;
+using Microsoft.ApplicationInsights.Extensibility;
+
+namespace Microsoft.Health.Fhir.Web
+{
+    public class CloudRoleNameTelemetryInitializer : ITelemetryInitializer
+    {
+        public void Initialize(ITelemetry telemetry)
+        {
+            telemetry.Context.Cloud.RoleName = "Microsoft Fhir Server";
```

Comment:
The name we have in the capability statement is _Microsoft FHIR Server_. Perhaps match this string initially (https://github.com/microsoft/fhir-server/blob/master/src/Microsoft.Health.Fhir.Core/Data/R4/BaseCapabilities.json)

Reply:
Done @brendankowitz.